client/man/Makefile.am

```diff
@@ -10,7 +10,8 @@ man1_MANS = 				\
 		ipa-client-install.1	\
 		ipa-client-automount.1	\
 		ipa-certupdate.1	\
-		ipa-join.1
+		ipa-join.1		\
+		ipa.1
```

Comment:
Either put the `ipa.1` packaging change in a separate commit, or don't do it at all (there must be some way to package custom file with `setuptool`, right?)